docs/source/deployment_guide_overview.md

```diff
@@ -0,0 +1,151 @@
+
+1. Edit the YAML file bundled with the binaries.
+2. Use environment variable overrides when deploying (if using Docker).
+
+The former option has the advantage of persisting your changes whenever you bring down and bring back up the node. The downside is that you will have to port the options you customized to the new YAML when upgrading to a new binary version (you should use the latest YAML when upgrading to a new version). By using a file like `./env<name of node>.list` that contains your environment variables, it is easy to port your changes to a new YAML.
+
+Either way, here are some values in `orderer.yaml` you might want to edit. You will notice that some of these fields are the same as those in `core.yaml`
+
+* `LocalMSPID`: this is the name of the local MSP, generated by your CA, of your orderer organization. This is where your orderer organization admins will be listed.
+
+* `LocalMSPDir`: the place where the local MSP for the ordering node is located. Note that it is a best practice to mount this volume external to your container.
+
+* `ListenAddress` and `ListenPort`: represents the endpoint to other ordering nodes in the same organization, an important consideration when establishing gossip communication among ordering nodes.
+
+* `# TLS`: this is where you specify whether you want to [enable TLS](enable_tls.html). If you set this value to `true`, you will have to specify the locations of the relevant TLS certificates. Note that all of the nodes in a network (both the peers and the ordering nodes, must either all have TLS enabled or not enable. For production networks, it is highly recommended to enable TLS). As with your MSP, it is a best practice to mount this volume external to your container.
+
+* `FileLedger`: although ordering nodes do not have a state database, they still all carry copies of the blockchain, as this allows them to verify permissions using the latest config block. Therefore the ledger fields should be customized with the correct endpoints.
+
+* `gossip`: there are a number of configuration options to think about when setting up [gossip](gossip.html), including the `externalEndpoint` (which makes peers discoverable to peers owned by other organizations) as well as the `bootstrap` address (which identifies a peer in the peer's own organization).\
+
+* `Cluster`: these values are important for ordering service nodes that communicate with other ordering nodes, such as in a Raft based ordering service.
+
+* `BootstrapFile`: this is the name of the genesis block you will generate for this ordering service.
+
+* `BootstrapMethod`: the method by which the bootstrap block is given. For now, this can only be `file`, in which the file in the `BootstrapFile` is specified.
+
+* `Consensus`: determines the key/value pairs allowed by the consensus plugin (Raft ordering services are supported and recommended) for the Write Ahead Logs (`WALDir`) and Snapshots (`SnapDir`).
+
```

Comment:
Again I would add operations service variables to this list.

Reply:
Same comment as above.